mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -758,6 +744,13 @@ func resourceComputeInstanceGroupManagerUpdate(d *schema.ResourceData, meta inte
 
 	d.Partial(false)
 
+	if d.Get("wait_for_instances").(bool) {
```

Comment:
To be equivalent to the old check in Read shouldn't we wait pre-update and pre-delete as well?

Reply:
that tracks for me